exec-ui/pom.xml

```diff
@@ -86,7 +86,7 @@
             </goals>
 <!-- Optional configuration which provides for running any npm command -->
             <configuration>
-              <arguments>install</arguments>
+              <arguments>install --unsafe-perms</arguments>
```

Comment:
Can you explain why you need this? is this something the user can override in `.npmrc`?

Reply:
This is internal used for dockers in T-Mobile, removed.